### DISCUSSION TITLE
Is Software renderer support rotation-angle?

### DISCUSSION THREAD

**Discussion by fluolite** [slint-ui/slint#7087] [General] [answered]
Is Software renderer support rotation-angle?

**Comment by tronical** [comment, marked as answer]
Unfortunately that's not supported right now. See also #3068

**Reply by fluolite** [reply]
Is there a plan to implement this?
 I think this is more important

**Reply by tronical** [reply]
We believe that it can be done, and we agree that it's generally useful. If you need the ability to rotate images with software rendering today, then an alternative that you could use is the Skia renderer.
